example/helloworld/Makefile

```diff
@@ -1,18 +1,19 @@
 RM = rm -f
 CC = gcc
 CFLAGS += -O2 -g -fpie
```

Comment:
Hum... since we are compiling an application (not a shared library) shouldn't `-fpie` be preferred? According to `man gcc`:
```
       -fpie
       -fPIE
           These options are similar to -fpic and -fPIC, but the
           generated position-independent code can be only linked into
           executables.  Usually these options are used to compile code
           that will be linked using the -pie GCC option.

           -fpie and -fPIE both define the macros "__pie__" and
           "__PIE__".  The macros have the value 1 for -fpie and 2 for
           -fPIE.
```

Reply:
Oh, I had no idea. Leave it like that.